Add files via upload

Neenjahw opened the pull request.

Neenjahw committed:
Add files via upload

So1ovyov commented on Sprint_04/MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -0,0 +1,195 @@
+    private var currentQuestionIndex = 0
+    private var correctAnswers = 0
+    
+    struct QuizStepViewModel {
```
Для этой структуры и двух следующих ты также можешь добавить `private`

So1ovyov commented on Sprint_04/MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -0,0 +1,195 @@
+            showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
+        }
+    }
+/*
```
Mock данные можно удалить, так как выше ты их уже добавил в массив

Neenjahw committed:
1)Исправлены шрифты кнопок 'Да' и 'Нет'
2)Добавлено скругление у постеров и цветной рамки
3)Изображения полностью заполняют UIImageView
4)Добавлено свойство isEnabled

Neenjahw committed:
Пропала часть кода, восстановил

Neenjahw reviewed (commented):
1

Neenjahw committed:
Merge remote-tracking branch 'refs/remotes/origin/sprint_04'
